hepcrawl/spiders/cds_spider.py

```diff
@@ -36,34 +40,29 @@ class CDSSpider(XMLFeedSpider):
     """
 
     name = 'CDS'
-    iterator = 'xml'
-    itertag = 'OAI-PMH:record'
-    namespaces = [
-        ('OAI-PMH', 'http://www.openarchives.org/OAI/2.0/'),
-        ('marc', 'http://www.loc.gov/MARC21/slim'),
-    ]
-
-    def __init__(self, source_file=None, **kwargs):
-        super(CDSSpider, self).__init__(**kwargs)
-        self.source_file = source_file
 
-    def start_requests(self):
-        yield Request(self.source_file)
+    def __init__(self, from_date=None, set="forINSPIRE", *args, **kwargs):
+        super(CDSSpider, self).__init__(url='http://cds.cern.ch/oai2d', metadata_prefix='marcxml', set=set, from_date=from_date, **kwargs)
```

Comment:
this is not very nice. I think it's better to have the same kind of API as [`XMLFeedSpider`](https://doc.scrapy.org/en/latest/topics/spiders.html#xmlfeedspider-example), just assigning some variables.